lib/ws_clients/hf_ds/on_rm_cores.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const { notifySuccess } = require('../../util/ws/notify')
+module.exports = (dsClient, msg) => {
+  const { d, requests, promises } = dsClient
+  const { reqID } = msg[1]
+  const ws = requests[reqID]
+
+  if (!ws) {
+    return d('recv man.rm cores for unknown req ID: %s', reqID)
+  }
+  notifySuccess(ws, 'Cores has been removed sucessfuly')
```

Comment:
the remove command returns an updated list of hypercores which can be used to update the ui, thats not sent to the client here: https://github.com/bitfinexcom/bfx-hf-data-server/blob/d4dee99df7f9ea199ddc2979adde3e00b65301da/lib/cmds/manage_cores.js#L49

from the description:

```
2. delete a list of hypercores: delete one or multiple hypercores.
after the command is finished, a list of the currently available
hypercores is sent to the client, which can be used for UI updates

```